db-server/index.js

```diff
@@ -177,6 +176,14 @@ function createServer(db) {
     }
   }
 
+  api.get('/account/:id/devices', withIdAndBody(db.accountDevices))
+
+  api.get(
+    '/account/:uid/device/:deviceId',
```

Comment:
Over in the auth-server, whenever we want to get the device record for a particular device, we get it for *all* devices and then filter down to just the one we want.  I added an explicit "get the record for one specific device" method here so we can avoid that in future.